lib/ringbuf/macros/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ringbuf-macros"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
+
+[features]
+# To disable a ring buffer (but leave it otherwise present), enable the
```

Comment:
Do you need this on the derive crate?

Reply:
ah, yeah, this is vestigial, good catch!